odata-data-aggregation-ext/7 Examples.md

```diff
@@ -1345,6 +1345,117 @@ ancestors(
+Content-Type: application/json
+
+{ "SuperordinateID": "EMEA Central" }
+```
+:::
+
+If the parent-child relationship between sales organizations is maintained in a separate entity set,
+the entity key may differ from the node identifier property and there can be entities without node identifier. And by using a [non-standard definition of start node](#RecursiveHierarchy), even nodes with node identifier can be unreachable from any root, these are called orphans.
+
+::: example
+⚠ Example ##ex_orphan: Assume additional `SalesOrganizations` Mars, Phobos and Venus, and that only Sales is a start node:
+```xml
+<EntityType Name="SalesOrganizationRelation">
+  <Key>
+    <PropertyRef Name="NodeID" />
+    <PropertyRef Name="SuperordinateID" />
+  </Key>
+  <Property Name="ID" Type="Edm.String" Nullable="false" />
+  <Property Name="SuperordinateID" Type="Edm.String" Nullable="false" />
+  <NavigationProperty Name="Superordinate"
+                      Type="SalesModel.SalesOrganization">
+    <ReferentialConstraint Property="SuperordinateID"
+                           ReferencedProperty="NodeID" />
+  </NavigationProperty>
+</EntityType>
+<EntityType Name="SalesOrganization">
+  <Key>
+    <PropertyRef Name="ID" />
+  </Key>
+  <Property Name="ID" Type="Edm.String" Nullable="false" />
+  <Property Name="Name" Type="Edm.String" />
+  <Property Name="NodeID" Type="Edm.String" />
+  <NavigationProperty Name="Relations"
+                      Type="Collection(SalesModel.SalesOrganizationRelation)">
+    <ReferentialConstraint Property="NodeID"
+                           ReferencedProperty="NodeID" />
+  </NavigationProperty>
+  <Annotation Term="Aggregation.RecursiveHierarchy"
+              Qualifier="MultiParentHierarchy">
+    <Record>
+      <PropertyValue Property="NodeProperty"
+                     PropertyPath="NodeID" />
+      <PropertyValue Property="ParentNavigationProperty"
+                     PropertyPath="Relations/Superordinate" />
+      <PropertyValue Property="IsStartNode">
+        <Eq>
+          <Path>ID</Path>
+          <String>Sales</String>
+        </Eq>
+      </PropertyValue>
+    </Record>
+  </Annotation>
+</EntityType>
+```
+
+Further assume the following relationships between sales organizations:
+
+NodeID|SuperordinateID
+------|---------------
+Sales|
+EMEA|Sales
+EMEA Central|EMEA
+Phobos|Mars
+Venus|
+
+Then the entity Mars has no node identifier and hence is no node in the hierarchy, even though it occurs as a parent. Its child Phobos cannot be reached from the root Sales and hence is an orphan, as is Venus.
+
+Mars and Phobos can be made descendants of the root node by adding a relationship. Note the first segment of the `ParentNavigationProperty` appears at the end of the resource path and the second segment appears in the payload before the `@bind`:
+```json
+POST /service/SalesOrganizations('Mars')/Relations
+Content-Type: application/json
+
+{ "Superordinate@bind": "SalesOrganizations('Sales')" }
```

Comment:
Is it possible to achieve the same effect with a request where both segments appear together?